modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,42 @@
+= Introduction to data migration
+:page-tag: migration,zdm,zero-downtime,zdm-proxy,introduction
+ifdef::env-github,env-browser,env-vscode[:imagesprefix: ../images/]
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration. 
+
+A migration is a workflow that encompasses the lifecycle of uploading and importing your data to the selected keyspaces. 
+DataStax has developed a set of thoroughly tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration
+* Cassandra Data Migrator
+* DSBulk Migrator
+ 
+With these tools, you can move your application to {astra_db}, DSE, or Cassandra with no downtime and with minimal configuration changes.
+Your clusters are kept in sync at all times by a dual-write logic configuration.
+You can roll back at any point, for complete peace of mind.
+
+== Migration process
+
+Downtime may be necessary when you need to migrate data that is critical for ongoing operations. 
```

Comment:
This should be worded as "If the business can tolerate some amount of application downtime and the data can be migrated within this period, tools such as Cassandra Data Migrator or DSBulk can be used for the migration."

Reply:
+1 to Faizal's suggestion